opendbc/dbc/generator/honda/_nidec_common.dbc

```diff
@@ -4,7 +4,8 @@ BO_ 145 KINEMATICS: 8 XXX
  SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" EON
 
 BO_ 148 KINEMATICS_ALT: 8 XXX
```

Comment:
I'm noticing that _nidec_common / KINEMATICS_ALT is the same thing as _bosch_2018 / KINEMATICS. I didn't notice this on my last cleanup sweep.

Can you send a separate clean PR to merge those as _honda_common / KINEMATICS and rename the other Nidec 145 KINEMATICS signal to KINEMATICS_ALT?

I'll merge that, then in this PR you can add your new KINEMATICS signal to the new common version. That way we don't start accumulating more generator drift/rot.

Reply:
Opened https://github.com/commaai/opendbc/pull/2797